blog/2023-05-11-conda-doctor

```diff
@@ -0,0 +1,37 @@
+---
+title: "conda doctor : Detecting conda environment inconsistencies easily"
+slug: "2023-05-11-conda-doctor"
+description: |
+    The new `conda doctor` command allows users to detect packages with missing files in their conda environments.
+
+authors: [forgottenprogramme]
+tags: [conda, environments]
+
+---
+The newest `conda doctor` command will be available with the latest conda 23.5.0, allowing conda users to detect corrupted packages (packages with files missing) in their conda environment quickly.
+
+One way packages in an environment may go corrupt is when one or more files in that package are accidentally deleted. Even a single missing file leads to a corrupted package.
+Until now there was no way to learn of these missing files except manually checking for their existence. With `conda doctor`, users can learn about all such packages in their conda environment with a single short command. 
+
+<add gif of conda doctor running>
+
+## How Does It Work?
+Every conda environment has a `conda-meta` directory associated with it, which keeps a log of all the packages (and all the files in them) ever installed into that environment. 
+`conda doctor` parses these metadata files, checks for the existence of each of the listed files and reports about the missing ones. 
+This is extremely helpful as it is otherwise very difficult to know that a package has missing files and to detect these missing files in the package. 
+
+## The Plugin Mechanism 
+`conda doctor` is implemented using conda’s new [plugins mechanism](https://www.anaconda.com/blog/introducing-a-new-plugin-mechanism-for-conda/), which means it has been implemented as an “add-on” without altering the core conda codebase.
```

Comment:
I would mention that it utilizes the "subcommand plugin hook" and that it comes bundled with conda itself. I would also avoid the term "add-on" as it might confuse people. Let's just stick with the term plugin.